codecov.yml

```diff
@@ -19,3 +19,6 @@ coverage:
 comment:
   layout: "header, diff"
   behavior: default
+
+ignore:
+  - package-lock.json
```

Comment:
I think you need to install a plugin: https://github.com/editorconfig/editorconfig-vscode